locales/hi.ftl

```diff
@@ -0,0 +1,20 @@
+title = Firefox Quantum Release कैंपेन
+project-name = Firefox Quantum Release
+main-title = Firefox स्ट्राइक्स बैक!
+main-description = नये Firefox के बारे में लोगो को बतायें। 14 नवम्बर को शाम साढ़े-सात बजे आइए ऑनलाइन और ट्वीट करिए अपनी मनपसंद राय नए Firefox के बारे में। साथ में #newfirefox लगाना बिलकुल मत भूलियेगा। आप Firefox के ट्वीट्स को रीट्वीट भी कर सकते हैं।
+thunderclap-cta = मदद करें Firefox Quantum की। 
+thunderclap-link = https://www.thunderclap.it/projects/64595-firefox-quantum-launch-day
+official-tweet-cta = रीट्वीट करें ऑफिसियल ट्वीट को
+tweet-1 = नए Firefox को उपयोग में लाएं और जाने की कैसे ये पहले से दो गुना ज्यादा तेज़ बन गया है। 💥 अभी Download करे : https://mzl.la/newfirefox
+tweet-2 = नया Firefox __________ तेज़ है ? डाउनलोड करें और जानें अभी: https://mzl.la/newfirefox । शुक्रीया @Firefox!
+tweet-3 = नया @Firefox आ चुका है, अब पहले से और भी ज्यादा सक्षम, बेहतर और तेज़।  कैसे? Download करिये और खुद जान लीजिये :  https://mzl.la/newfirefox  
```

Comment:
I'm missing the hashtag here. Could you please add it so we can measure our impact?

Reply:
Hi Sambhav, I think their needs some change in Hindi translation. Ex: line number 7 should be like this : आधिकारिक ट्वीट को रीट्वीट करें.